hangman/lib/hangman/application.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Hangman.Application do
```

Comment:
# -3

So I may be wrong (in which case let me know and I'll add the 4 back) but this seems to be a supervisor that simply supervises another supervisor. I don't see what that gets you. Couldn't you just have directly invoked the Hangman.Supervisor module from here?